examples/gameroom/gameroom-app/src/store/api.ts

```diff
@@ -146,7 +146,7 @@ export async function listGames(circuitID: string): Promise<Game[]> {
 export async function submitPayload(payload: Uint8Array): Promise<void> {
   const options = {
     method: 'POST',
-    url: `http://${window.location.host}/api/submit`,
+    url: `${document.location.protocol}//${window.location.host}/api/submit`,
```

Comment:
Does this work if you just start the string with `/api/submit` or does it require a complete URL